.travis.yml

```diff
@@ -127,8 +127,10 @@ matrix:
 install:
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
+    - pip install https://github.com/keflavich/httpbin/archive/master.zip
```

Comment:
although it may be better not listing it in ``CONDA_DEPENDENCIES``, as it may mess up things resulting the conda installed version being picked up rather than the pip installed.